waku/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -701,7 +701,10 @@ proc getInt(
   try:
     retInt = parseInt(str)
   except ValueError:
-    return err("exception in getInt, parseInt: " & getCurrentExceptionMsg())
+    return err(
+      fmt"exception in getInt, parseInt, str:[{str}] query:[{query}]: " &
```

Comment:
FYI `fmt` can raise and prevent using `raises: []` on procs.

Reply:
> FYI `fmt` can raise and prevent using `raises: []` on procs.

Thanks for the comment! I fixed that even though I cannot link to the actual commit (I had to `force-push` as I rebased from origin.)